src/stores/bridge.ts

```diff
@@ -66,8 +67,8 @@ export default class Bridge extends ContractStore {
   @observable public slots: IObservableArray<Slot> = observable.array([]);
   @observable public lastCompleteEpoch: number;
 
-  constructor(account: Account, address?: string) {
-    super(bridgeAbi, address);
+  constructor(account: Account, transactions: Transactions, address?: string) {
```

Comment:
Here `transactions` in other places `txs`. Let's use one of that, not both

Reply:
good catch!